.buildkite/pipeline-resource-definitions/kibana-artifacts-snapshot.yml

```diff
@@ -21,7 +21,6 @@ spec:
       env:
         ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'true'
       allow_rebuilds: true
-      branch_configuration: main 8.14 7.17
```

Comment:
needs to be removed, or replaced with a generic `*.*` + `main` type expression, because we don't want to keep updating this in every version switch